src/components/UserSelector.tsx

```diff
@@ -73,7 +56,7 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(setAuthor(user));
```

Comment:
Also, remember to prevent the default action of the click event to avoid any unexpected behavior due to the event bubbling up. You can do this by calling `event.preventDefault()` in the `onClick` handler.